### PR TITLE
Implemented the main spaces page, only shows current listings

### PR DESCRIPTION
**Pull request by APWilson97**
Only shows current listings on the database in line with MVP specs

### CODE REVIEW

File: templates/spaces/new.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            </p>
+
+            <p>
+            <label for="price">Price Per Night:</label><br>
+            <input type="number" name="price_per_night" id="price">
+            </p>
+
+            <p>
+            <label for="email">Email Address:</label><br>
+            <input type="text" name="email_address" id="email_address">
```
Comment (elliepriestley) [inline templates/spaces/new.html]: We could make the input type for email_address an email, for automatic validation purposes

File: templates/spaces/new.html
Comment (elliepriestley) [inline templates/spaces/new.html:1]: Please use indents for paragraph tags etc

**Review by elliepriestley — CHANGES_REQUESTED** [review]
Lovely work - just a few suggestions

File: templates/spaces/new.html (hunk 1 above)
Comment (APWilson97) [reply templates/spaces/new.html]: Changed input type to email

File: templates/spaces/new.html
Comment (APWilson97) [reply templates/spaces/new.html:1]: Added indentation to label and input tags within paragraph tags

**Review by elliepriestley — APPROVED** [review]
Reviewed all requested changes - looks great

**Pull request merged**